packages/core/src/Stores/traders-hub-store.js

```diff
@@ -365,6 +365,14 @@ export default class TradersHubStore extends BaseStore {
                 : localize(
                       'Trade CFDs on MT5 with forex, stocks, stock indices, synthetics, cryptocurrencies, and commodities.'
                   );
+        const swapfree_account_desc =
+            !this.is_eu_user || this.is_demo_low_risk
+                ? localize(
+                      'Trade CFDs on MT5 with synthetics, forex, stocks, stock indices, cryptocurrencies and ETFs swap-free.'
+                  )
+                : localize(
+                      'Trade CFDs on MT5 with forex, stocks, stock indices, synthetics, cryptocurrencies, and commodities.'
+                  );
```

Comment:
```suggestion
        const swapfreeAccountDeesN = () => (
             !this.is_eu_user || this.is_demo_low_risk
                ? localize(
                      'Trade CFDs on MT5 with synthetics, forex, stocks, stock indices, cryptocurrencies and ETFs swap-free.'
                  )
                : localize(
                      'Trade CFDs on MT5 with forex, stocks, stock indices, synthetics, cryptocurrencies, and commodities.'
                  );
        );
```
[Fantastic Comment #1] Things with conditional `localize` like this one needs to be wrapped in a function if they are in a component. There is a chance that this text does not get re-rendered when the language is changed

Reply:
done